dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/nonfinancials/EuTaxonomyDetailsPerCashFlowType.kt

```diff
@@ -12,14 +9,18 @@ import java.math.BigDecimal
  */
 data class EuTaxonomyDetailsPerCashFlowType(
     val totalAmount: DataPointOneValue<AmountWithCurrency>?,
-    val totalNonEligibleShare: RelativeAndAbsoluteFinancialShare?,
-    val totalEligibleShare: RelativeAndAbsoluteFinancialShare?,
-    val totalNonAlignedShare: RelativeAndAbsoluteFinancialShare?,
+    val nonEligibleShare: RelativeAndAbsoluteFinancialShare?,
+    val eligibleShare: RelativeAndAbsoluteFinancialShare?,
+    val nonAlignedShare: RelativeAndAbsoluteFinancialShare?,
     val nonAlignedActivities: List<EuTaxonomyActivity>?,
-    val totalAlignedShare: RelativeAndAbsoluteFinancialShare?,
-    @field:Schema(example = JsonExampleFormattingConstants.SUBSTANTIAL_CONTRIBUTION_CRITIREA)
-    val substantialContributionCriteria: Map<EnvironmentalObjective, BigDecimal>?,
+    val alignedShare: RelativeAndAbsoluteFinancialShare?,
+    val substantialContributionToClimateChangeMitigation: BigDecimal?,
```

Comment:
Marc had a good remark here: Should we add "percentage" to the names of these fields, as these will hold percentage values.

Reply:
This has been done now.

Reply:
ok